### ISSUE TITLE
Leftover tree inventory

### ISSUE THREAD

**Issue by christophe-rd** [christophe-rd/fuelinex#16]
This is a quick issue to keep track of how many trees are left from Pheanoflex and Fuelinex as of today (June 24, 2024). Note: I selected only the trees that were doing good and excluded the ones that were borderline dead. 

#### Deciduous
Acer Negundo: 88 
Alnus Rubra: 90
Acer Macrophyllum: 12
Populus Balsamifera : 102 (although they are already very tall)

#### Evergreen 
Thuja Occidentalis : 8
Sequoiadendron Giganteum: 52
Pinus Contarta: 49
Pinus Strobus: 86

**Comment by christophe-rd** [comment]
@lizzieinvancouver thanks!
Yes, I agree for the 3 species you mentionned.

For Populus, I think they will be to big. We could still take some cuttings out of them.

I think it would be kinda sad to send them for compost. Do you think we could give/sell them?

**Comment by lizzieinvancouver** [comment]
> I think it would be kinda sad to send them for compost. Do you think we could give/sell them?

@christophe-rd Yes, definitely! Do whatever you can to move them a better place. We could also post them as available for experiments on a slack or such (I say this as someone without slack).
